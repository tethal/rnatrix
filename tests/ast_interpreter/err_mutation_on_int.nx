fun main() {
    var x = 42;
    x[0] = 1;
}