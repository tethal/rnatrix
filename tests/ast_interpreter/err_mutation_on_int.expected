<string>:3:5: error: only lists support indexing in assignments
    x[0] = 1;
    ^^^^
